architecture/daisy/README.md

```diff
@@ -2,34 +2,52 @@
+It is recommended to put your Faust DSP files inside a directory [DaisyExamples](https://github.com/electro-smith/DaisyExamples/)`/DaisySP/faust_examples`. Then execute `faust2daisy code.dsp` with your chosen options. To use `faust2daisy` in an arbitrary directory, you should specify environment variables. For example, on macOS/Linux:
+```bash
+export LIBDAISY_DIR=~/GitHub/DaisyExamples/libdaisy
+export DAISYSP_DIR=~/GitHub/DaisyExamples/DaisySP
+```
+If on macOS, consider putting the above text in `~/.zshrc` so that it's always set in Terminal.
+
+The default optimization is for file size: `OPT=-Os`. You can optimize for speed by setting `OPT=-O2` or the even more aggressive setting `OPT=-O3`.
 
-The current **faust2daisy** tool can only be used to program the [POD](https://www.electro-smith.com/daisy/pod). On this board, the 2 *switches* and 2 *knobs* can be connected to UI controllers using metadata:
+## Daisy Pod
 
-- `[switch:N]` (with N from 1 to 2) has to be used in a `button` or `checkbox` item to connect it to the POD switch number N.
-- `[knob:N]` (with N from 1 to 2) has to be used in a `vslider`, `hslider` or `nentry` item to connect it to the POD knob number N. The knob [0..1] range will be mapped to the slider/nentry [min..max] range.
+The **faust2daisy** tool can be used to program the [Pod](https://www.electro-smith.com/daisy/pod). On this board, the 2 *switches* and 2 *knobs* can be connected to UI controllers using metadata:
+
+- `[switch:N]` (with N from 1 to 2) has to be used in a `button` or `checkbox` item to connect it to the Pod switch number N.
+- `[knob:N]` (with N from 1 to 2) has to be used in a `vslider`, `hslider` or `nentry` item to connect it to the Pod knob number N. The knob [0..1] range will be mapped to the slider/nentry [min..max] range.
 
 Other metadata:
 
-- `[scale:lin|log|exp]` metadata is implemented.
+- `[scale:lin|log|exp]` metadata is implemented for knobs.
+
+## Daisy Patch
 
-## Bootloader configuration:
+The **faust2daisy** tool can be used to program the [patch.Init()](https://electro-smith.com/products/patch-init).
+
+## Bootloader Configuration:
 
 Setup the board to utilize the manufacturer's bootloader, ensuring that the generated code fits within the board's memory. For detailed instructions on flashing the bootloader, please refer to: [Daisy Bootloader Getting Started Guide](https://electro-smith.github.io/libDaisy/md_doc_2md_2__a7___getting-_started-_daisy-_bootloader.html).
 
-## DSP examples
+## Memory Limits (SDRAM)
+
+If you're using a Daisy with significant amount of SDRAM such as the Pod or patch.Init, then you may want to use the `-sdram` flag when compiling code involving long delay lines/tables etc. Enabling this will execute a Python script inside `faust2daisy` that modifies the generated C++ code to put large float buffers (e.g., delay lines) in SDRAM. For this to work, you must have `python3` available on the command line.
```

Comment:
Do you think a custom memory manager could be used here instead of modifying the generate C++ code ? (see https://faustdoc.grame.fr/manual/architectures/#custom-memory-manager)

Reply:
Yea I know that using Python here is not the Faust way, but I'm just trying to get a working solution before an upcoming workshop. We can implement something better later.

Reply:
OK